pkg/lightning/common/pause_test.go

```diff
@@ -39,19 +39,24 @@ func (checker *unblocksChecker) Check(params []interface{}, names []string) (boo
 	max := params[2].(time.Duration)
 
 	ch := make(chan time.Duration)
+	start := time.Now()
 	go func() {
-		start := time.Now()
 		wg.Wait()
 		ch <- time.Since(start)
 	}()
 	select {
 	case dur := <-ch:
 		if dur < min {
-			return false, "WaitGroup unblocked before minimum duration"
+			return false, "WaitGroup unblocked before minimum duration, it was " + dur.String()
 		}
 		return true, ""
 	case <-time.After(max):
-		return false, "WaitGroup did not unblock after maximum duration"
+		select {
+		case dur := <-ch:
+			return false, "WaitGroup did not unblock after maximum duration, it was " + dur.String()
+		case <-time.After(1 * time.Second):
```

Comment:
why wait one more second?

Reply:
If we meet `WaitGroup did not unblock after maximum duration` again, we can get the real time it takes to unblock. This might would be useful to find the failure was caused by meaning test requirement or some thing worse(like, deadlock).